test/test_with_postgresql.py

```diff
@@ -118,6 +118,7 @@ def test_date_str_dmy(self):
         # self._test_layer(self.layer, DATE_STR_COL_DMY,timevectorlayer.DateTypes.DatesAsStrings,
         #                 CUSTOM_FORMAT_DMY)
 
+    @unittest.skip
     def test_date(self):
         self._test_layer(self.layer, DATE_COL, timevectorlayer.DateTypes.DatesAsStrings,
                          time_util.DEFAULT_FORMAT)
```

Comment:
How so? time_util.DEFAULT_FORMAT doesn't have the T in it. Are you running the tests on an otherwise clean branch (no other changes)?